app/controllers/moments_controller.rb

```diff
@@ -63,7 +62,8 @@ def delete_comment
     end
 
     if comment_exists && ((is_my_comment && is_a_viewer) || is_my_moment)
-      delete_comment_and_notifications(params[:commentid], 'moment')
+      CommentNotifications.remove(comment_id: params[:commentid],
```

Comment:
Heh, I didn't know you could do that in Ruby. TIL :)

Reply:
Is this some kind of dependency injection?